pkg/configuration/parameters.go

```diff
@@ -127,6 +131,10 @@ type JiraApiResourceParametersEditCustomField struct {
 	CustomFieldValueFromFile *string
 }
 
+type JiraApiResourceParametersAddComment struct {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [type JiraApiResourceParametersAddComment should be JiraAPIResourceParametersAddComment](https://app.codacy.com/manual/TurnsCoffeeIntoScripts/jira-api-resource/pullRequest?prid=5025522)